spec/expression.dd

```diff
@@ -1906,6 +1906,16 @@ $(GNAME ValueExpression):
         a $(I ValueSeq), then the elements of the $(I ValueSeq)
         are inserted as arguments in place of the sequence.
     )
+    
+    $(P If there are duplicate keys in the $(I KeyValuePairs), then
+        the value returned for that key is going to be that of the
+        last $(I KeyValuePair) declared.
```

Comment:
I think it's worthwhile to mention that the duplicates are not actually stored, rather that the value for a specific key is updated to the last assigned value. Also, it's important to note that duplicates can be present in the AA initializer, but only the last lexicographically declared variable is stored.